src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Requests/GeneratePreviewReportRequest.cs

```diff
@@ -7,6 +7,23 @@
 
 namespace Microsoft.SqlTools.ServiceLayer.TableDesigner.Contracts
 {
+    /// <summary>
+    /// The service request to generate preview report describing the changes.
+    /// </summary>
+
+    public class GeneratePreviewReportResult 
+    {
+        /// <summary>
+        /// The report generated in plaintext or markdown
```

Comment:
I'd just remove the "plaintext or markdown" stuff in these comments. Just keeping this generic is fine - it's up to the callee to handle the various mime types appropriately but we shouldn't be specifying here what those types might be since we support sending anything.